README.md

```diff
@@ -1,77 +1,107 @@
 # Deis Workflow v2
 
-[![Build Status](https://travis-ci.org/deis/workflow.svg?branch=master)](https://travis-ci.org/deis/workflow) [![codecov.io](https://codecov.io/github/deis/workflow/coverage.svg?branch=master)](https://codecov.io/github/deis/workflow?branch=master)
+[![Build Status](https://travis-ci.org/deis/workflow.svg?branch=master)](https://travis-ci.org/deis/workflow)
+[![codecov.io](https://codecov.io/github/deis/workflow/coverage.svg?branch=master)](https://codecov.io/github/deis/workflow?branch=master)
+[![Docker Repository on Quay](https://quay.io/repository/deisci/workflow/status "Docker Repository on Quay")](https://quay.io/repository/deisci/workflow)
```

Comment:
Thanks - fixed in https://github.com/deis/workflow/commit/c0a96bdb997d2235f1e4bd9cd34858588efe55f8